pydicom/util/fixes.py

```diff
@@ -0,0 +1,137 @@
+"""Compatibility fixes for older version of python."""
+
+# Authors : Guillaume Lemaitre <g.lemaitre58@mgail.com>
+# Copyright 2008-2017 pydicom authors. See LICENSE file for details.
+
+import sys
+
+from datetime import datetime, tzinfo, timedelta
+
+if sys.version_info[0] < 3:
+    class timezone(tzinfo):
+        """Backport of datetime.timezone.
+
+        Notes
+        -----
+        Backport of datetime.timezone for Python 2.7, from Python 3.6
+        documentation (https://tinyurl.com/z4cegu9), copyright Python Software
+        Foundation (https://docs.python.org/3/license.html)
+
+        """
+        __slots__ = '_offset', '_name'
+
+        # Sentinel value to disallow None
+        _Omitted = object()
+
+        def __new__(cls, offset, name=_Omitted):
+            if not isinstance(offset, timedelta):
+                raise TypeError("offset must be a timedelta")
+            if name is cls._Omitted:
+                if not offset:
+                    return cls.utc
+                name = None
+            elif not isinstance(name, str):
+                raise TypeError("name must be a string")
+            if not cls._minoffset <= offset <= cls._maxoffset:
+                raise ValueError("offset must be a timedelta "
+                                 "strictly between -timedelta(hours=24) and "
+                                 "timedelta(hours=24).")
+            if (offset.microseconds != 0 or offset.seconds % 60 != 0):
+                raise ValueError("offset must be a timedelta "
+                                 "representing a whole number of minutes")
+            return cls._create(offset, name)
+
+        @classmethod
+        def _create(cls, offset, name=None):
+            self = tzinfo.__new__(cls)
+            self._offset = offset
+            self._name = name
+            return self
+
+        def __getinitargs__(self):
+            """pickle support"""
+            if self._name is None:
+                return (self._offset,)
+            return (self._offset, self._name)
+
+        def __eq__(self, other):
+            if type(other) != timezone:
+                return False
```

Comment:
I would assume that the Python guys thought that triggering True is actually the only test which cover all the case. We can make another test to cover the line but it would be only to make the line green.